api/base/serializers.py

```diff
@@ -79,22 +79,26 @@ def _tpl(val):
     return None
 
 
+#todo probably needs refactoring
 def _get_attr_from_tpl(attr_tpl, obj):
-    attr_name = _tpl(str(attr_tpl))
-    if attr_name:
-        attribute_value = getattr(obj, attr_name, ser.empty)
-        if attribute_value is not ser.empty:
-            return attribute_value
-        elif attr_name in obj:
-            return obj[attr_name]
-        else:
-            raise AttributeError(
-                '{attr_name!r} is not a valid '
-                'attribute of {obj!r}'.format(
-                    attr_name=attr_name, obj=obj,
-                ))
+    if hasattr(obj, 'pk'):
```

Comment:
It is being used by the link class right now.  It grabs an attribute name and returns it's value.  I was using it because it was there and already handy but I can look into a more efficient way of doing this if you want